pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -448,7 +448,7 @@ func writeConfig(content []byte, fullFilename string) error {
 func writeSecret(content []byte, fullFilename string) error {
 	klog.Infof("Writing secret manifest %q ...", fullFilename)
 
-	filePerms := os.FileMode(0600)
```

Comment:
I don't think this should change.  I'm pretty sure I explicitly made this 0600.

Reply:
@deads2k Git blame says it's Polynomial that made it 600 on 6/8/21

Reply:
@EmilyM1 hmm, I did some more history digging and found the original change being done here: https://github.com/openshift/library-go/pull/391/files. So this has already been changed from 0644 to 0600 to limit read/write to the root user.

Looking at https://bugzilla.redhat.com/show_bug.cgi?id=1977730 I think there is a different ask: `/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/user-serving-cert-000/tls.crt` and `/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/user-serving-cert-000/tls.key` currently have 0644 permissions, the expected permissions should be 0600.

I am not sure if those assets are handled in this `writeSecret` function as well, but if not probably a different spot needs a fix to set permissions to 0600 on user-provided serving certs and keys.

Reply:
I don't see why we would want 0644, definitely not for TLS keys.

Reply:
@s-urbaniak  @stlaz:  when keys are placed in a secret yaml -- isn't it actually k8's itself that sets the permissions on this file?
I am no longer sure if this is something we can fix.
The closest I could see of a way to change it was our writeSecret func.

Reply:
Updated to 0600 per discussions